src/MobileHeader.jsx

```diff
@@ -1,12 +1,16 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
+import { injectIntl, intlShape } from 'react-intl';
 
 // Local Components
 import { Menu, MenuTrigger, MenuContent } from './Menu';
 import Avatar from './Avatar';
 import { LinkedLogo } from './Logo';
 
+// i18n
+import messages from './messages';
```

Comment:
Did we want to follow the same conventions as in Profile, so the messages for Foo.jsx go in Foo.messags.jsx ?

Reply:
I was thinking that since this is a single component and in this particular case Desktop and Mobile Headers will use mostly the same strings keeping it in one place may be better. wdyt?